src/app/games/games.service.ts

```diff
@@ -85,6 +85,11 @@ export class GamesService {
         });
         return this.http.request(request).do(() => {
             this.toastyService.success('Player added!');
+        }).catch(err => {
+            if (err.status === 409) {
+                this.toastyService.warning('Player is already in game');
```

Comment:
You can provide some additional endpoint to check if user si in game/team. I can call it on FE in case there is an additional "targetTeam"/"targetGame" parameter in user profile component.